lib/src/locale/locale_parser.dart

```diff
@@ -0,0 +1,210 @@
+class LocaleParser {
+  // gathered from wikipedia
+  static final Map<String, Map<String, String>> _isoLanguages = {
```

Comment:
niestety dart nie ma wbudowanego tego w framework, a poslugiwanie sie 3rd party libkami zwieksza ryzyko niekompatybilnosci. Na tą chwilę to rozwiązanie jest najbardziej lightweight